rtf_vis_tool/src/tests/SE3.test.js

```diff
@@ -21,13 +21,13 @@ describe('SE3 Class Unit Tests', () => {
                                    [1, 0, 0]]);
 
         const translation = nj.array([2,3,4]);
-        const testSE3 = SE3(rotation, translation);
+        const testSE3 = new SE3(rotation, translation);

```

Comment:
is `new` required here? heap allocation?

Reply:
yes, it was throwing `TypeError: Class constructor SE3 cannot be invoked without 'new'` error otherwise

Reply:
yes, it was throwing `TypeError: Class constructor SE3 cannot be invoked without 'new'` error otherwise

Reply:
> is `new` required here? heap allocation?

yes, it was throwing `TypeError: Class constructor SE3 cannot be invoked without 'new'` error otherwise